pages/documentation/simulation/powerflow/index.md

```diff
@@ -119,34 +128,34 @@ The default value is `false`.
 The `writeSlackBus` is an optional property that says if the slack bus has to be written in the network using the [slack terminal extension](../../grid/model/extensions.md#slack-terminal) after a load flow computation.
 The default value is `false`.
 
-**noGeneratorReactiveLimits**  
-The `noGeneratorReactiveLimits` property is an optional property that defines whether the load flow is allowed to find a setpoint value outside the reactive limits of a generator or not.
-The default value is `false`.
+**useReactiveLimits**  
+The `useReactiveLimits` property is an optional property that defines whether the load flow should take into account equipment's reactive limits. Applies to generators, batteries, SVCs, dangling lines, and HVDC VSCs.
+The default value is `true`.
 
 **phaseShifterRegulationOn**  
-The `phaseShifterRegulationOn` property is an optional property that defines whether the load flow is allowed to change taps of a phase tap changer or not.
+The `phaseShifterRegulationOn` property is an optional property that defines whether phase shifter regulation should be enabled in the load flow.
```

Comment:
Regulation is not my favorite term for that (even if it is used in the parameter name). Control is more clear for me, but it is up to you.